component/audioinput/v1/audioinput.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
+	v1_0 "go.viam.com/api/common/v1"
```

Comment:
why did switch from calling it v1 to v1_0 here, but in other files we went from calling it v1_0 to v1?

Reply:
Removed from PR